Soccar [Game-Server]/src/nl/soccar/gameserver/controller/socnet/message/handler/BallSyncMessageHandler.java

```diff
@@ -0,0 +1,30 @@
+package nl.soccar.gameserver.controller.socnet.message.handler;
+
+import io.netty.buffer.ByteBuf;
+import nl.soccar.gameserver.controller.socnet.message.BallSyncMessage;
+import nl.soccar.socnet.connection.Connection;
+import nl.soccar.socnet.message.MessageHandler;
+
+public final class BallSyncMessageHandler extends MessageHandler<BallSyncMessage> {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)